src/pages/index.js

```diff
@@ -69,25 +117,40 @@ const previewModalDeleteBtn = previewModal.querySelector(
 
 const previewModalErrorMsg = previewModal.querySelector(".modal__error");
 
+// CARD RELATED ELEMENTS //
+
 const cardTemplate = document.querySelector("#card-template");
 const cardsList = document.querySelector(".cards__list");
 
+let selectedCard, selectedCardId;
+
 function getCardElement(data) {
   const cardElement = cardTemplate.content
     .querySelector(".card")
     .cloneNode(true);
 
   const cardNameEl = cardElement.querySelector(".card__title");
   const cardImageEl = cardElement.querySelector(".card__image");
+
   const cardLikeBtn = cardElement.querySelector(".card__like-btn");
 
+  console.log(cardElement);
+
   //TODO - Select the delete button
   const cardDeleteBtn = cardElement.querySelector(".card__delete-btn");
 
+  // TODO ... if the card is liked, set the active class on the card, liked cards should be liked when page is refreshed
+
   cardNameEl.textContent = data.name;
   cardImageEl.src = data.link;
   cardImageEl.alt = data.name;
 
+  if (data.isLiked) {
+    cardLikeBtn.classList.add("card__like-btn_liked");
+  } else {
+    cardLikeBtn.classList.remove("card__like-btn_liked");
+  }
+
   cardLikeBtn.addEventListener("click", () => {
     cardLikeBtn.classList.toggle("card__like-btn_liked");
```

Comment:
`    cardLikeBtn.classList.toggle("card__like-btn_liked");
`


Please, delete this click handler because you handle likes in line 168 with `changeLikeStatus`


You need to **toggle likes** only in a `then` block after a successful response from the server because if there is an error, the user will not know about that and will see the toggled heart color.